### ISSUE TITLE
Complete provisional model for NEURIPS proposal

### ISSUE THREAD

**Issue by ChristinaLast** [AQ-AI/open-geo-engine#62]
- [x] Get locations of low cost sensors
- [x] query satellite data for train/test set
  - [x] get satellite imagery for training set (sensor locations in Dehli)
